akka-testkit/src/main/scala/akka/testkit/TestKit.scala

```diff
@@ -811,8 +863,10 @@ trait TestKitBase {
  *
  * @since 1.1
  */
+@deprecated("Use akka.testkit.scaladsl.TestKit instead", since = "2.5.0")
 class TestKit(_system: ActorSystem) extends { implicit val system = _system } with TestKitBase
 
+@deprecated("Use akka.testkit.scaladsl.TestKit instead", since = "2.5.0")
```

Comment:
yeah. I will put them in scaladsl and deprecated the old one including `TestProbe`, `ImplicitSender`, `DefaultTimeout`. Since the `TestProbe` is used in java test, a javadsl version is needed too.